CONTRIBUTING.md

```diff
@@ -76,25 +76,31 @@ Check out our dedicated [Bevy Organization document](/docs/the_bevy_organization
 
 ### Classifying PRs
 
-Our merge strategy relies on the classification of PRs on two axes:
-
-* How controversial are the design decisions.
-* How complex is the implementation.
-
-Each [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
-
-* A: Area (e.g. A-Animation, A-ECS, A-Rendering).
-* C: Category (e.g. C-Breaking-Change, C-Code-Quality, C-Docs).
-* D: Difficulty (e.g. D-Complex, D-Good-First-Issue).
-* O: Operating System (e.g. O-Linux, O-Web, O-Windows).
-* P: Priority (e.g. P-Critical, P-High).
-* S: Status (e.g. S-Blocked, S-Controversial, S-Needs-Design).
-
-PRs with non-trivial design decisions are given the [`S-Controversial`] label. This indicates that
-the PR needs more thorough design review or an [RFC](https://github.com/bevyengine/rfcs), if complex enough.
-
-PRs that are non-trivial to review are given the [`D-Complex`] label. This indicates that the PR
-should be reviewed more thoroughly and by people with experience in the area that the PR touches.
+[Labels](https://github.com/bevyengine/bevy/labels) are our primary tool to organize work.
+Each label has a prefix denoting its category:
+
+* **D:** Difficulty. In order, these are:
+  * `D-Trivial`: typos, obviously incorrect one-line bug fixes, code reorganization, renames
+  * `D-Straightforward`: simple bug fixes and API improvements, docs, test and examples
+  * `D-Modest`: new features, refactors, challenging bug fixes
+  * `D-Complex`: rewrites and unusually complex features
+  * When applied to an issue, these labels reflect the estimated level of expertise (not time) required to fix the issue.
+  * When applied to a PR, these labels reflect the estimated level of expertise required to *review* the PR.
+  * The `D-Domain-Expert` and `D-Domain-Agnostic` labels are modifiers, which describe if unusually high or low degrees of non-Rust knowledge are required.
+* **X:** Controversiality. In order, these are:
+  * `X-Uncontroversial`: everyone should agree that this is a good idea
+  * `X-Contentious`: there's real design thought needed to ensure that this is the right path forward
+  * `X-Controversial`: there's active disagreement and/or large-scale architectural implications involved
+* **A:** Area (e.g. A-Animation, A-ECS, A-Rendering).
+* **C:** Category (e.g. C-Breaking-Change, C-Code-Quality, C-Docs).
+* **O:** Operating System (e.g. O-Linux, O-Web, O-Windows).
+* **P:** Priority (e.g. P-Critical, P-High)
+  * Most work is not explicitly categorized by priority: volunteer work mostly occurs on an ad hoc basis depending on contributor interests
+* **S:** Status (e.g. S-Blocked, S-Needs-Review, S-Needs-Design).
+
+The rules for how PRs get merged depend on their classification by controversy and difficulty.
+More difficult PRs will require more careful review from experts,
+while more controversial PRs will require rewrites to reduce the costs involved and/or sign-off from Subject Matter Experts and Maintainers.
```

Comment:
Nit: Should we link to the [SME](https://github.com/bevyengine/bevy/blob/3cca08ecfe15609746a2beb01655ffd548898ecb/docs/the_bevy_organization.md#subject-matter-expert-sme) page?

Reply:
I think that's not worth it here. When we split this out into a proper book I'll do a pass on cross-linking though.